src/inverseRobot.js

```diff
@@ -18,13 +18,25 @@
  * inverseRobot(robert) === null
  * inverseRobot(kolli) === { name: 'Kolli', chipVer: '123', wheels: '3' }
  *
- *
- * @param {object} robot
- *
- * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const inverseObject = {};
+  const values = Object.values(robot);
+
+  for (let i = 0; i < values.length; i++) {
+    for (let j = i + 1; j < values.length; j++) {
+      if (values[i] === values[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
You don't need this, just check if inverseObject has key you're trying to add in loop below and return null if it does